### PR TITLE
Update conflicts episode

### PR DESCRIPTION
**Pull request by chrissyrissmeyer**
Closes: #23

### CODE REVIEW

File: _episodes/10-conflicts.md [hunk 1]
```diff
@@ -9,528 +9,407 @@ objectives:
...
+
+Conflicts can also be minimized with project management strategies:
+
+- Clarify who is responsible for what areas with your collaborators
+- Discuss what order tasks should be carried out in with your collaborators so
+  that tasks expected to change the same lines won't be worked on simultaneously
+- Design your workflow to minimize the potential for conflicts.
+- If the conflicts are stylistic (e.g. tabs vs. spaces), establish a
+  project convention that is governing and use code style tools (e.g.
+  `htmltidy`, `perltidy`, `rubocop`, etc.) to enforce, if necessary
```
Comment (dunn) [inline _episodes/10-conflicts.md]: Was there supposed to be more to this sentence?

**Review by dunn — APPROVED** [review]

File: _episodes/10-conflicts.md (hunk 1 above)
Comment (chrissyrissmeyer) [reply _episodes/10-conflicts.md]: that's all there was in the original

Comment (dunn) [reply _episodes/10-conflicts.md]: Oh gotcha.  Then add a period and we're good to go!

Comment (chrissyrissmeyer) [reply _episodes/10-conflicts.md]: hahahaha the only one that had a period was the sentence i added (Design your workflow to minimize the potential for conflicts.). i'll add to all the others and update the pr

**Comment by dunn** [comment]
Oh, I didn't realize that was a bullet point, it was such a large paragraph.

But this is better, thanks!

**Review by mcritchlow — APPROVED** [review]
👍

**Pull request merged**